### DISCUSSION TITLE
Is it advised to rebuild all antlr4 generated files when version was updated?

### DISCUSSION THREAD

**Discussion by duc-gp** [antlr/antlr4#3122] [Q&A] [answered]
assumption:
- we have build the files from SearchGrammer with antlr4 4.9.1 with
`antlr4 -Dlanguage=JavaScript /path/to/SearchGrammar2.g4`

happening:
- now in our package json we have updated to antlr4 4.9.2

question:
- should we now get the latest antlr4 command line jar (https://www.antlr.org/download/antlr-4.9.2-complete.jar) and rebuild with the `antlr4 -Dlanguage=JavaScript /path/to/SearchGrammar2.g4` command?

**Comment by ericvergnaud** [comment, marked as answer]
yes

**Reply by duc-gp** [reply]
okay, thanks
